gulpfile.js

```diff
@@ -96,9 +97,11 @@ gulp.task("yarn:build", () => {
 /**
  * Run the test task
  */
-gulp.task("yarn:test", () => {
+gulp.task("jest:test", () => {
```

Comment:
this is duplicating the test logic (`yarn test` will also run `ts-lint`), if you want to move the `test` logic here, you should remove the `test` script from `package.json` to avoid confusion (and also add linting)